packages/react/src/popover-anchor/PopoverAnchor.tsx

```diff
@@ -0,0 +1,20 @@
+import * as RadixPopover from "@radix-ui/react-popover";
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+
+type PopoverAnchorProps = BoxProps<typeof RadixPopover.Root>;
```

Comment:
Vai, Will it have the same props of Root or Anchor?

Reply:
fixed